### PR TITLE
Refactor directory and namespace

### PR DESCRIPTION
**Pull request by at-wat**

### CODE REVIEW

**Comment by at-wat** [comment]
@nhatao please take a look

File: src/urg_stamped_node.cpp [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2018-2019 The urg_stamped Authors
```
Comment (nhatao) [inline src/urg_stamped_node.cpp:2]: 2018-2021

File: include/urg_stamped/urg_stamped.h [hunk 2]
```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright 2018-2019 The urg_stamped Authors
```
Comment (nhatao) [inline include/urg_stamped/urg_stamped.h:2]: ditto

**Review by nhatao — APPROVED** [review]
LGTM
It would be better to update years in copyrights.

**Pull request merged**